ios/utils/NSDataAdditions.h

```diff
@@ -0,0 +1,10 @@
+//
+//  NSDataAdditions.h
+//
+//  Created by Till POS on 14/09/21.
+//
+
+@interface NSData ()
```

Comment:
What is the purpose of this header and implementation?

Reply:
Converting base64 to binary.